### ISSUE TITLE
Dropdown (search selection) not binding model to controller property

### ISSUE THREAD

**Issue by allthesignals** [Semantic-Org/Semantic-UI-Ember#111]
Ember: 1.13
Semantic-UI-Ember: 0.9.3

I may just be misunderstanding these release notes:
https://github.com/Semantic-Org/Semantic-UI-Ember/pull/46

But the module doesn't seem to be binding the selected object to the controller property.

Handlebars:
```
{{#ui-dropdown class="search selection" value=organization2}}

  {{input type="text" class="search" autocomplete="off" tabindex=0 value=searchCriteria}}
  <div class="default text">Search and Select</div>
  <i class="dropdown icon"></i>
  <div class="menu">
    {{#each organizations as |organizationChoice|}}
      {{!-- <div class="item" data-id="{{organizationChoice.id}}" data-value="{{organizationChoice.id}}"> --}}
        {{#ui-dropdown-item content=organizationChoice}}
          {{organizationChoice.name}}
        {{/ui-dropdown-item}}
      {{!-- </div> --}}
    {{/each}}
  </div>
{{/ui-dropdown}}
```

The `organizations` property is a computed property returning an enumerable Ember model:
```
  organizations: function() {
    return this.store.query("organization", { "filter[search]": this.get("searchCriteria") });
  }.property("this.searchCriteria"),
```

Sorry if this is obvious, I'm just not sure I'm understanding the documentation. Using onChange and setting the property that way works correctly, however, I'd like to get the more concise approach to work if possible.

Thank you

**Comment by ivanbanov** [comment]
+1

**Comment by kcollignon** [comment]
+1

**Comment by aaronbhansen** [comment]
We have released the 2.0 beta branch, which totally re-worked the ui-dropdown and bound values. Please upgrade and test against the latest branch.
